config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-presubmit-dra-capz.yaml

```diff
@@ -320,12 +320,16 @@ presubmits:
           value: "3s"
         - name: CL2_LONG_JOB_RUNNING_TIME
           value: "150m"
+        - name: CL2_PROMETHEUS_CPU_SCALE_FACTOR
+          value: "2"
+        - name: CL2_PROMETHEUS_MEMORY_SCALE_FACTOR
+          value: "10"
```

Comment:
I was aiming to hit 20Gi with this based on the template: https://github.com/kubernetes/perf-tests/blob/a0680b203a3b3e125380c16950ab55a0e7a23eba/clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml#L28-L42